### ISSUE TITLE
Add GIF of CML rainfall animation to README

### ISSUE THREAD

**Issue by cchwala** [OpenSenseAction/OPENSENSE_sandbox#63]
The nice new animation from #62 should be added as GIF to the README as a teaser for the sandbox content.

**Comment by cchwala** [comment]
I recorded a screencast of the animation and converted it to GIF via
```
 ffmpeg -i OPENSENSE_sandbox_OpenMRG_usecase_animation.mov OPENSENSE_sandbox_OpenMRG_usecase_animation.gif
```

Here it is 🎉 

![OPENSENSE_sandbox_OpenMRG_usecase_animation](https://github.com/OpenSenseAction/OPENSENSE_sandbox/assets/102827/58594da1-350c-46d7-936c-049608086121)

**Comment by cchwala** [comment]
Closed via #65
